README.md

```diff
@@ -260,10 +260,49 @@ Check out our [TensorFlow.js demo](https://github.com/mindee/doctr-tfjs-demo) to
 
 ### Docker container
 
-If you wish to deploy containerized environments, you can use the provided Dockerfile to build a docker image:
+[We offers Docker container support for easy testing and deployment](https://github.com/mindee/doctr/packages).
+
+#### Using GPU with docTR Docker Images
+
+The docTR Docker images are GPU-ready and based on CUDA `11.8`.
+However, to use GPU support with these Docker images, please ensure that Docker is configured to use your GPU.
+
+To verify and configure GPU support for Docker, please follow the instructions provided in the [NVIDIA Container Toolkit Installation Guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
+
+Once Docker is configured to use GPUs, you can run docTR Docker containers with GPU support:
+
+```shell
+docker run --it --gpus all ghcr.io/mindee/doctr:tf-py3.8.18-v0.7.1 bash
+```
+
+#### Available Tags
+
+The Docker images for docTR follow a specific tag nomenclature: `<framework>-py<python_version>-<doctr_version|YYYY-MM>`. Here's a breakdown of the tag structure:
+
+- `<framework>`: `tf` (TensorFlow) or `torch` (PyTorch).
+- `<python_version>`: `3.8.18`, `3.9.18`, or `3.10.13`.
+- `<doctr_version>`: a tag >= `v0.7.1`
+- `<YYYY-MM>`: e.g. `2023-10`
+
+Here are examples of different image tags:
+
+| Tag                        | Description                                       |
+|----------------------------|---------------------------------------------------|
+| `tf-py3.8.18-v0.7.1`       | TensorFlow version `3.8.18` with docTR `v0.7.1`. |
+| `torch-py3.9.18-2023-10`| PyTorch version `3.9.18` with a monthly build from `2023-10`. |
+
```

Comment:
Maybe additional to the badge link to the site where we can see all available images :)

Reply:
@odulcy-mindee ignore this comment we have it already in Line 263